### ISSUE TITLE
Conflicts with B9 and Interstellar fuel switches.

### ISSUE THREAD

**Issue by allista** [allista/ConfigurableContainers#11]
I just did a test and here's what I observed:

* Started a new sandbox game in a clean install, no mods.
* Installed Nertea's Cryogenic Engines, which bundles B9PartSwitch and patches the stock fuel tanks to be able to hold LH2.
* In the VAB, configured a stock FL-T200 fuel tank to hold LH2+O instead of LF+O, and placed it on the * launchpad with a probe core on top.
* Quit the game and installed ConfigurableContainers.
* Went back to the fuel tank on the launchpad.  It now had an empty resource bar for LiquidFuel, in addition to the full ones for LqdHydrogen and Oxidizer.
* Opened CC's "edit tanks" window.  It only showed capacity for LiquidFuel and Oxidizer, no LqdHydrogen, despite the tank's right-click menu showing LqdHydrogen actually in it.
* Quit the game and deleted ConfigurableContainers.
* Went back to the fuel tank on the launchpad.  Now it had only the LiquidFuel and Oxidizer resource bars — the LqdHydrogen had vanished.

Bottom line: CC has its own built-in idea of what resource types a tank holds by default, and it does weird things if some other fuel switcher has already changed what a particular tank holds.  I don't want it doing those weird things in my career save, where I have vessels in space that've been configured with other fuel switchers.

**Comment by allista** [comment]
Should be fixed in v1.2.0
